.github/workflows/docker.yaml

```diff
@@ -0,0 +1,147 @@
+name: docker
+
+on:
+  push:
+    branches:
+      - "**"
+    tags:
+      - "v*.*.*"
```

Comment:
Aha, this makes me realise. My default choice would be to have separate version history for the API and UI. They would produce their own tags like `@cube-creator/core-api@v0.1.0`.

Unless you would like a single version for the entire repository @martinmaillard ?

Reply:
I would personally have opted for a single version for the entire repository, as it makes maintenance easier. Any argument for having separate versions?

Reply:
Building and deploying separately. Many changes in the UI will be unrelated to the API - no version bump. And vice versa.

I'm not sure that a single version is going to make reduce maintenance

Reply:
Building and deploying will be automated so I don't see a problem deploying everything for every version, even if one of the parts didn't change.

Reply:
@tpluscode ok for you if we go with a single version?

Reply:
Yes, tentatively. I would like to see a proposal for managing this version. As far as tooling is concerned there is nothing that I know of which will help us.

To explain, we have a monorepo which is at least the UI and API which we can evolve somewhat independently by keeping their change history and versions. Tooling exists to manage this (`lerna version` or `atlassian changesets`)

The alternative is what? To ignore the `version` in packages and only keep the one in root `package.json`?

Reply:
Unless we publish the API and the UI as packages I'm not sure it really matters. What's important at this point is having docker images to deploy

Reply:
We package them as two docker images which is close enough, isn't it?

Reply:
Right, I missed that part. Let's go with this and we'll revisit when we need to.